GramAddict/core/bot_flow.py

```diff
@@ -147,7 +149,7 @@ def start_bot():
                 running_ig_version = get_instagram_version()
                 logger.info(f"Instagram version: {running_ig_version}")
                 if tuple(running_ig_version.split(".")) > tuple(
-                    TESTED_IG_VERSION.split(".")
+                        TESTED_IG_VERSION.split(".")
```

Comment:
can you black using the default settings?

Reply:
i format it with black